src/components/AuthModal/AuthModal.test.tsx

```diff
@@ -21,11 +21,11 @@ describe('AuthModal', () => {
 
     await render(<AuthModal isOpen onClose={jest.fn} />);
 
-    expect(screen.getByRole('heading', { name: /please install supported wallet/i })).toBeInTheDocument();
+    expect(screen.getByRole('heading', { name: /please install a supported wallet/i })).toBeInTheDocument();
```

Comment:
minor, but if we're checking for text strings we should use a dictionary file so that we don't need to change the test file if the dictionary entry changes. (Although that assumes the string is in the dictionary—we have a lot of instances where the copy is in the component).

No need to do this now though—our current translations file needs to be re-considered, so this is something we can take care of at that point.